bundles/org.openhab.binding.mybmw/pom.xml

```diff
@@ -14,4 +14,171 @@
 
   <name>openHAB Add-ons :: Bundles :: MyBMW Binding</name>
 
+  <profiles>
```

Comment:
@wborn - can you help review the changes in this file?

Reply:
Sorry, I missed that. Thanks for the explanation.

Reply:
Can I keep it, would be good

Reply:
Let's keep it then if we don't receive any feedback from @wborn before the PR is ready to be merged (which seems to be quite soon).